shell/browser/native_window.h

```diff
@@ -147,6 +147,7 @@ class NativeWindow : public base::SupportsUserData,
   virtual bool IsExcludedFromShownWindowsMenu() = 0;
   virtual void SetSimpleFullScreen(bool simple_fullscreen) = 0;
   virtual bool IsSimpleFullScreen() = 0;
+  virtual void UpdateFrame() = 0;
```

Comment:
minor style nit: This can be behind [#if defined(OS_MAC)](https://github.com/electron/electron/pull/28150/files#diff-bcde70ba560ca4f196ebb71ba43f10d2551c112219bb3474647ee50312b74f17R205) below which avoids having to implement noop for other platforms.